ccl/streamingccl/streamingest: TestTenantStreamingCutoverOnSourceFailure failed

ccl/streamingccl/streamingest.TestTenantStreamingCutoverOnSourceFailure [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8046655?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8046655?buildTab=artifacts#/) on master @ [10266a323f94c3cf397d5de590e512d987e63e22](https://github.com/cockroachdb/cockroach/commits/10266a323f94c3cf397d5de590e512d987e63e22):


```
=== RUN   TestTenantStreamingCutoverOnSourceFailure
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingCutoverOnSourceFailure647287828
    test_log_scope.go:79: use -show-logs to present logs inline
    stream_replication_e2e_test.go:964: error executing 'RESUME JOB 824233434248249345': pq: job 824233434248249345: job with status succeeded cannot be resumed
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingCutoverOnSourceFailure647287828
--- FAIL: TestTenantStreamingCutoverOnSourceFailure (5.27s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/tenant-streaming
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingCutoverOnSourceFailure.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-22652

--- a/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
+++ b/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
@@ -583,9 +583,6 @@ func TestTenantStreamingCutoverOnSourceFailure(t *testing.T) {
 	c.DestSysSQL.Exec(c.T, `ALTER TENANT $1 COMPLETE REPLICATION TO SYSTEM TIME $2::string`,
 		c.Args.DestTenantName, cutoverTime.AsOfSystemTime())
 
-	// Resume ingestion.
-	c.DestSysSQL.Exec(t, fmt.Sprintf("RESUME JOB %d", ingestionJobID))
-
 	// Ingestion job should succeed despite source failure due to the successful cutover
 	jobutils.WaitForJobToSucceed(t, c.DestSysSQL, jobspb.JobID(ingestionJobID))
 }

